index.js

```diff
@@ -418,6 +418,13 @@ class HyperBee {
     return iteratorToStream(new DiffIterator(new Batch(this, null, false, opts), new Batch(right, null, false, opts), opts), active)
   }
 
+  async hasPrefix (prefix) {
```

Comment:
@andrewosh do you have a better name for this?

Reply:
checkPrefix ?